test/Model/CommentSpec.hs

```diff
@@ -0,0 +1,46 @@
+module Model.CommentSpec
+    ( main
+    , spec
+    ) where
+
+import TestImport
+import Model.Comment
+import Data.Time.Clock (addUTCTime)
+
+main :: IO ()
+main = hspec spec
+
+spec :: Spec
+spec = withApp $
+    describe "deleteStaleComments" $
+        it "deletes comments older than a week for the given site" $ do
+            siteId <- createSiteWithUrl "http://example.com/site"
+            otherSiteId <- createSiteWithUrl "http://example.com/otherSite"
+            Entity u _ <- runDB $ createUser "example"
+            runDB $ do
+                setCreatedAgeInDays 6 =<<
+                    createComment u siteId "x" "x" "recent"
```

Comment:
I've slowly extracting a generict `buildX` out of all the `createX` functions we have currently. This could be a good oppurtunity to make `insert $ buildComment { commentCreate = x, commentBody = "recent" }` work...